trex-scripts/tests/priority_scheduling.py

```diff
@@ -0,0 +1,152 @@
+
+    # setup_subparser is an optional class method
+    # You can implement this method if you want to add additional command line
+    # parameters for your test.
+    # Those parameters will be parsed and be passed to the "start" method below as
+    # "args" argument.
+    @classmethod
+    def setup_subparser(cls, parser: ArgumentParser) -> None:
+        parser.add_argument("--duration", type=float, help="Test duration", default=10)
+
+
+    # The entrypoint of a test
+    def start(self, args: dict) -> None:
+
+        # create packets
+        pkt1 = Ether(dst=DEST_MAC) / IP(src="16.0.0.1",dst="48.0.0.1") / UDP(dport=12,sport=1025) / ("*" * 1500)
+        pkt2 = Ether(dst=DEST_MAC) / IP(src="16.0.0.2",dst="48.0.0.2") / UDP(dport=12,sport=1025) / ("*" * 1500)
+        pkt3 = Ether(dst=DEST_MAC) / IP(src="16.0.0.3",dst="48.0.0.3") / UDP(dport=12,sport=1025) / ("*" * 1500)
+
+        #stream list
+        streams = []
+
+        # Create a traffic stream
+        # assume s1 is a delay critical stream with QoS
+        s1 = STLStream( packet=STLPktBuilder(pkt=pkt1), mode=STLTXCont(percentage = 1), flow_stats = STLFlowLatencyStats(pg_id = 5))
+        # assume s2 is a delay critical stream without QoS
+        s2 = STLStream( packet=STLPktBuilder(pkt=pkt2), mode=STLTXCont(percentage = 1), flow_stats = STLFlowLatencyStats(pg_id = 10))
+        # assume s3 is a lower priority stream
+        s3 = STLStream( packet=STLPktBuilder(pkt=pkt3), mode=STLTXCont(percentage = 98), flow_stats = STLFlowLatencyStats(pg_id = 15))
+
+
+        # prepare ports
+        self.client.reset(ports=[0,1])
+
+        # add sterams
+        streams.append(s1)
+        streams.append(s2)
+        streams.append(s3)
+        self.client.add_streams(streams, ports=[0])
+
+
+        logging.info(
+                "Starting traffic, duration: %d sec", args.duration,
+        )
+
+        # Start sending traffic
+        self.client.start(SENDER_PORT, mult="100%", duration=args.duration)
+        pgids = self.client.get_active_pgids()
+
+
+        logging.info("Waiting until all traffic stop")
+        self.client.wait_on_traffic(ports=SENDER_PORT)
+
+
+        # stats for pg_id 5 and 10
+        stats              = self.client.get_pgid_stats(pgids['latency'])
+        flow_stats_5       = stats['flow_stats'].get(5)
+        flow_stats_10      = stats['flow_stats'].get(10)
+        global_lat_stats   = stats['latency']
+        lat_stats_5        = global_lat_stats.get(5)
+        lat_stats_10       = global_lat_stats.get(10)
+
+        tx_pkts_5  = flow_stats_5['tx_pkts'].get(0, 0)
+        rx_pkts_5  = flow_stats_5['rx_pkts'].get(1, 0)
+        drops_5    = lat_stats_5['err_cntrs']['dropped']
+
+        tx_pkts_10  = flow_stats_10['tx_pkts'].get(0, 0)
+        rx_pkts_10  = flow_stats_10['rx_pkts'].get(1, 0)
+        drops_10    = lat_stats_10['err_cntrs']['dropped']
+
+        print(" \n TX and RX flow stats and packet dopped for pg id 5 (i.e., delay critical): ")
```

Comment:
Same here, let's use stream numbers in printed messages
```suggestion
        print(" \n TX and RX flow stats and packets dopped for s1 (i.e., delay critical): ")
```